### DISCUSSION TITLE
How to hide duplicate maven modules in Tycho workspace?

### DISCUSSION THREAD

**Discussion by Bananeweizen** [eclipse-tycho/tycho#416] [Q&A] [answered]
I have lived with these duplicate folders for nested modules for years now. And today I finally found the "Hide modules" filter of the package explorer. Is that how you all work with the Tycho workspace? If so, I would add it to the contributions doc. Or is there some other trick to avoid having duplicate files all over the place (there is also an experimental option in the maven workspace preferences, but that doesn't seem to work).

**Comment by mickaelistria** [comment, marked as answer]
I personally never use the Package Explorer, but the Project Explorer instead. By default it shows the projects hierarchically, if not you need to enable hierarchical view https://www.eclipse.org/eclipse/news/4.5/platform.php#nested-projects
Ctrl+3 de-duplicates resource by default, Ctrl+Shift+R can be configured to de-duplicate, Ctrl+Shift+T doesn't show duplicate... The only one that's still bugging me in the Search results which still show duplicate resources, or shows a resources from a parent project when I'd rather have the one from the child project.

**Comment by laeubi** [comment]
I also always use the `Project Explorer `and hierarchical project view (afaik usage of `Package Explorer `is discouraged).
Maybe it would be a good idea to share a screenshot of one of these duplicated things and we can check how to avoid them? 
Maybe related to this as well:
- https://github.com/eclipse-m2e/m2e-core/issues/342
